.github/workflows/Jirabot.yml

```diff
@@ -24,79 +24,113 @@ jobs:
           python -m site
           python -m pip install --upgrade pip setuptools wheel
           python -m pip install --upgrade jira
+          python -m pip --version
+          python -m pip freeze | grep jira
       - name: "Run"
         env:
           JIRABOT_USERNAME : ${{ secrets.JIRABOT_USERNAME }}
           JIRABOT_PASSWORD : ${{ secrets.JIRABOT_PASSWORD }}
-          JIRA_URL : ${{ secrets.JIRA_URL }}
+          JIRA_URL : ${{ vars.JIRA_URL }}
           PULL_REQUEST_NUMBER : ${{ github.event.pull_request.number }}
           PULL_REQUEST_TITLE : ${{ github.event.pull_request.title }}
           PULL_REQUEST_AUTHOR_NAME : ${{ github.event.pull_request.user.login }}
           PULL_URL: ${{ github.event.pull_request.html_url }}
           COMMENTS_URL: ${{ github.event.pull_request.comments_url }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
+          GHUB_JIRA_USER_MAP: ${{ vars.GHUB_JIRA_USER_MAP }}
         run: |
             import os
             import re
+            import json
             from jira.client import JIRA
 
+            def updateIssue(jira, issue, prAuthor: str, pull_url: str) -> str:
+                result = ''
+
+                statusName = str(issue.fields.status)
+                if statusName == 'Open':
+                    transition = 'Start Progress'
+                elif statusName == 'In Progress':
+                    transition = ''
+                elif statusName == 'Resolved':
+                    transition = 'Reopen Issue'
+                elif statusName == 'Closed':
+                    transition = 'Reopen Issue'
+                else:
+                    transition = ''
```

Comment:
Let's log this else condition, something like "Unexpected Jira status encountered: statusName"